diff -urN sdlmame-0.148/makefile sdlmame-0.148-patched/makefile
--- sdlmame-0.148/makefile	2013-01-14 02:19:22.744542314 +1100
+++ sdlmame-0.148-patched/makefile	2013-01-14 02:20:58.786228628 +1100
@@ -314,13 +314,13 @@
 endif
 
 # compiler, linker and utilities
-AR = @ar
-CC = @gcc
-LD = @g++
+AR = ar
+CC = gcc
+LD = g++
 MD = -mkdir$(EXE)
-RM = @rm -f
-OBJDUMP = @objdump
-PYTHON = @python
+RM = rm -f
+OBJDUMP = objdump
+PYTHON = python
 
 
 #-------------------------------------------------
@@ -826,12 +826,12 @@
 
 $(OBJ)/%.lh: $(SRC)/%.lay $(FILE2STR_TARGET)
 	@echo Converting $<...
-	@$(FILE2STR) $< $@ layout_$(basename $(notdir $<))
+	$(FILE2STR) $< $@ layout_$(basename $(notdir $<))
 
 $(OBJ)/%.fh: $(SRC)/%.png $(PNG2BDC_TARGET) $(FILE2STR_TARGET)
 	@echo Converting $<...
-	@$(PNG2BDC) $< $(OBJ)/temp.bdc
-	@$(FILE2STR) $(OBJ)/temp.bdc $@ font_$(basename $(notdir $<)) UINT8
+	$(PNG2BDC) $< $(OBJ)/temp.bdc
+	$(FILE2STR) $(OBJ)/temp.bdc $@ font_$(basename $(notdir $<)) UINT8
 
 $(DRIVLISTOBJ): $(DRIVLISTSRC)
 	@echo Compiling $<...
