diff -urN sdlmame-0.150/src/osd/osdcomm.h sdlmame-0.150-patched/src/osd/osdcomm.h
--- sdlmame-0.150/src/osd/osdcomm.h	2013-09-17 08:47:04.000000000 +1100
+++ sdlmame-0.150-patched/src/osd/osdcomm.h	2013-09-17 23:50:39.302016809 +1100
@@ -65,7 +65,7 @@
 #define ATTR_MALLOC             __attribute__((malloc))
 #define ATTR_PURE               __attribute__((pure))
 #define ATTR_CONST              __attribute__((const))
-#define ATTR_FORCE_INLINE       __attribute__((always_inline))
+#define ATTR_FORCE_INLINE
 #define ATTR_NONNULL(...)       __attribute__((nonnull(__VA_ARGS__)))
 #define ATTR_DEPRECATED         __attribute__((deprecated))
 /* not supported in GCC prior to 4.4.x */
