diff -urN sdlmame-0.151/src/osd/sdl/sdlmain.c sdlmame-0.151-patched/src/osd/sdl/sdlmain.c
--- sdlmame-0.151/src/osd/sdl/sdlmain.c	2013-11-03 02:42:34.000000000 +1100
+++ sdlmame-0.151-patched/src/osd/sdl/sdlmain.c	2013-11-08 16:01:24.604141871 +1100
@@ -498,7 +498,7 @@
 	MACRO_VERBOSE(LSB_FIRST);
 	MACRO_VERBOSE(PTR64);
 	MACRO_VERBOSE(MAME_DEBUG);
-	MACRO_VERBOSE(NO_DEBUGBER);
+	MACRO_VERBOSE(NO_DEBUGGER);
 	MACRO_VERBOSE(BIGENDIAN);
 	MACRO_VERBOSE(CPP_COMPILE);
 	MACRO_VERBOSE(DISTRO);
diff -urN sdlmame-0.151/src/osd/sdl/sdl.mak sdlmame-0.151-patched/src/osd/sdl/sdl.mak
--- sdlmame-0.151/src/osd/sdl/sdl.mak	2013-11-05 07:59:52.000000000 +1100
+++ sdlmame-0.151-patched/src/osd/sdl/sdl.mak	2013-11-08 16:00:54.398653215 +1100
@@ -528,8 +528,6 @@
 QT_INSTALL_HEADERS = $(shell $(QMAKE) -query QT_INSTALL_HEADERS)
 INCPATH += -I$(QT_INSTALL_HEADERS)/QtCore -I$(QT_INSTALL_HEADERS)/QtGui -I$(QT_INSTALL_HEADERS)
 LIBS += -L$(shell $(QMAKE) -query QT_INSTALL_LIBS) -lQtGui -lQtCore
-else
-DEBUGOBJS = $(SDLOBJ)/debugwin.o $(SDLOBJ)/dview.o $(SDLOBJ)/debug-sup.o $(SDLOBJ)/debug-intf.o
 endif
 
 LIBGL = -lGL
